src/nvim/os/win_defs.h

```diff
@@ -5,6 +5,10 @@
 #include <sys/stat.h>
 #include <stdio.h>
 
+#if !defined(MAXNAMLEN)
```

Comment:
Does this wrapper need to be here? `MAXNAMLEN` won't be defined on Windows AFAIK and the only reason I added this was because it was in `os_defs.h` at the time. In equalsraf@5a6b312 `MAXNAMLEN` is just defined directly.